docs/stepbystep/react/lesson1.md

```diff
@@ -0,0 +1,81 @@
+# Step by step apps creation with ILC, React, Lesson 1
+
+Hi folks! This document will teach you how to create simple isomorphic React application and run it with ILC in few minutes.
+
+So buckle in your seat belt and let's get started 🚀
+
+## We need React app with SSR first
+
+As you might already guess to build new micro frontend we need to have regular application first. 
+Hopefully I already did all the dirty job for you dear reader. 
+
+So you can go and grab it here https://github.com/StyleT/ilc-learning-react.
+Or you can use https://codesandbox.io/s/github/StyleT/ilc-learning-react to have it running for you in the cloud.
+
+As soon as you started the app - try to play few tic-tac-toe matches 😎
+
+## Adapting the app to ILC
+
+To make the application work correctly with ILC we need to make it compliant with [ILC to App interface](../../ilc_app_interface.md).
+Fortunately that's pretty easy, go through the source code and resolve all the `//TODO:` comment I placed for you there.
```

Comment:
in ilc-learing-react/server.js `// TODO:` contains space. remove it please, because I haven't found this todo during global search in the project

Reply:
fixed